README.md

```diff
@@ -262,6 +262,14 @@ To check whether dnsmasq name server at 127.0.0.1 can resolve subdomains of dev.
 app.dev.gov.uk.		0	IN	A	127.0.0.1
 ```
 
+### How to: work with local gems
+
+Most govuk-docker services have been configured to mount `~/govuk` at `/govuk`, so you can reference local applications like so:
+
+```ruby
+gem 'govuk_publishing_components', path: '/govuk/govuk_publishing_components' 
```

Comment:
The alternative I use is `path: '../govuk_publishing_components'`

Reply:
My `govuk` folder is in my home directory, so I use either:

`gem 'govuk_publishing_components', path: '~/govuk/govuk_publishing_components'`
Or
gem 'govuk_publishing_components', path: '../govuk_publishing_components'

Reply:
Can't use `~/govuk/govuk_publishing_components`, as `/govuk` is not the home directory in Docker. End up getting this error when trying to `govuk-docker build`:

```
The path `/home/build/govuk_publishing_components` does not exist.
```

`../govuk_publishing_components` works though, and is probably better because that works for both Docker and VM environments. 👍

Reply:
Thanks for approving @benthorner - I've just pushed up a tweak in response to this thread, could you re-read and re-approve please?